x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -163,6 +167,15 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
       },
       sortable: true,
     },
+    {
+      'data-test-subj': 'mlAnomaliesListColumnAction',
+      name: i18n.translate('xpack.ml.anomaliesTable.actionsColumnName', {
+        defaultMessage: 'Actions',
+      }),
+      render: (changePoint: ChangePoint) => {
+        return <LinksMenu changePoint={changePoint} dataViewId={dataViewId} />;
```

Comment:
As discussed, as there's only one action for now, you can just use the standard EUI table actions so that the Discover link is visible in the column, and no need for the sub menu.

Reply:
Updated in 0c14669335f451c5a2623f91b9b65158005da401